team.go

```diff
@@ -149,6 +160,49 @@ func (s *teams) Read(ctx context.Context, teamID string) (*Team, error) {
 	return t, nil
 }
 
+// TeamUpdateOptions represents the options for updating a team.
+type TeamUpdateOptions struct {
+	// For internal use only!
+	ID string `jsonapi:"primary,teams"`
+
+	// New name for the team
+	Name *string `jsonapi:"attr,name,omitempty"`
+
+	// The team's organization access
+	OrganizationAccess *OrganizationAccess `jsonapi:"attr,organization-access,omitempty"`
+}
+
+func (o TeamUpdateOptions) valid() error {
+	if !validString(o.Name) {
+		return errors.New("name is required")
+	}
```

Comment:
Are you sure `Name` is required? That would mean that if you want to update the org access, you also have to fill in the existing name?

Reply:
No, not required, my bad.